src/js/tabs/firmware_flasher.js

```diff
@@ -1224,7 +1224,7 @@ TABS.firmware_flasher.initialize = function (callback) {
     });
 };
 
-TABS.firmware_flasher.cleanup = function (callback) {
+firmware_flasher.cleanup = function (callback) {
```

Comment:
Just for me to understand, do we follow C coding style where the function opening bracket should be on the next line?
https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md

Reply:
No

Reply:
and if we do, that's something what should be included in linter. And everything would need to be rewritten.

Reply:
I think that's what I been told to do by @mikeller some time ago.
I understand lots of code already has been written the other way, but there is some other stuff that's against coding style. It does not mean we need to rewrite it right now.

Reply:
that style of braces etc is not that common in JS world. So don't really see why we need to follow it.

As for not doing it everywhere, it will just lead to what we have now. Bunch of different standards and so called lava pattern. If we'd decide to strictly stick to one, it would have to be 1 PR with all the changes and CI/CD rule to prevent other styles. Otherwise it will be 100 + 1 different way to write code.

Reply:
We should stick with the opening bracket on the same line. (But that's just my opinion)

Reply:
We should just agree on something :) But who is to give a final word?

Reply:
I like the way it's done here as most of the code does it this way :smiley: So agree with @chmelevskij here.